packages/terra-search-field/src/terra-dev-site/test/search-field/SearchFieldWithPlaceholder.test.jsx

```diff
@@ -2,4 +2,4 @@ import React from 'react';
 import SearchField from '../../../SearchField';
 
 export default () =>
-  <SearchField id="searchfield" placeholder="Search Text" />;
+  <SearchField id="searchfield" placeholder="Search Text" inputAttributes={{ 'aria-label': 'Search Field' }} />;
```

Comment:
Should remove these custom input attributes then?

Reply:
I should and have now done so. I left the Jest test in there, as I feel it still has use in testing overwriting the default value.